### DISCUSSION TITLE
Table of Custom Query doesn't update after data change

### DISCUSSION THREAD

**Discussion by wipplingermax** [Budibase/budibase#11130] [Help] [answered]
Hey folks,

im struggling with the following behaviour:

I've got a Table showing data of a custom query. 
By clicking on the table a modal screen opens.
On this screen the underlying data will be updated.
After updating and closing the modal screen the inital data won't update.

Data changes are written to the database.
Only refreshing the page would solve the problem. But im unable to refresh the inital page.

Thanks in advance!

**Comment by ducnhoxlatui** [comment]
hmmm, maybe you can add the refresh button to refresh the data provider of Table

**Comment by melohagan** [comment, marked as answer]
Hey @wipplingermax 

There's a related issue here: https://github.com/Budibase/budibase/issues/10399

Side panels could be an alternative option to a modal.
